website/docs/tutorials/07-testing/03-fuzzing/README.md

```diff
@@ -116,6 +123,22 @@ Protostar will run any test that takes parameters in fuzz testing mode, so let's
 test:
```

Comment:
This sentence is not reworked properly

Reply:
Sorry but I don't get which sentence, can you specify? You commented on an unchanged line preceded by unchanged text.

Reply:
>Protostar will run any test that takes parameters in fuzz testing mode, so let's rewrite our unit
test:

Reply:
I see, I just misunderstood, what you meant was "not reworked at all but it should've been"

Reply:
[docs updates](https://github.com/software-mansion/protostar/pull/995/commits/4521eb3d29990ea5d54633e16da1a8aa36b9c511)